examples/simple/README.md

```diff
@@ -0,0 +1,82 @@
+# Jupyter Server Simple Extension Example
+
+This folder contains an example of 2 simple extensions on top of Jupyter Server.
+
+## Install
+
+You need `python3` to build and run the server extensions.
+
+```bash
+conda create -y -n jext python=3.7
+conda activate jext
+pip install -e .
+```
+
+**OPTIONAL** If you want to build the Typescript code, you need `npm` on your local env. Compiled javascript is provided as artifact in this repository, so this Typescript build step is optional. The Typescript source and configuration has been taken from https://github.com/markellekelly/jupyter-server-example.
+
+```bash
+npm install
+npm run build
+```
```

Comment:
This throws an error because the `/src` directory is missing.

Reply:
src is now added. Thx Zach, Eric